health/api/v1/health.py

```diff
@@ -29,12 +30,19 @@ def get_blueprints():
     ]
 
 
+CACHE = cache.SimpleCache(threshold=50, default_timeout=30)
+
+
 @health.route("/", defaults={"region": "all"})
 @health.route("/<region>")
 def get_health(region):
 
     period = flask.request.args.get("period", "day")
 
+    result = CACHE.get("{}:{}".format(region, period))
```

Comment:
So I would like to see here some more senior approach. 

First of all this should be a decorator. 
Decorator should be unified and allow you to specify what arguments from request string to use to build key, by default it should use whole URL path for example (/region/?preiod=...). 

And as it is now common for any flask route we should make it part of our library "flask_helpers" and make it available for all services.